### DISCUSSION TITLE
Clicking the source tag on a beatmap page should search for the tag specifically in osu!direct

### DISCUSSION THREAD

**Discussion by Spok5508** [ppy/osu#30181] [Ideas] [answered]
Currently. clicking the source tag opens osu!direct with the search term being this tag. Since this is a loose query, you will find a lot of different beatmaps that might have a similar word in the title or artist. When searching for the tag with quotation marks for example, the search is a lot more precise and the user can quickly find the relevant beatmaps with the same source.

In the example below, the user is currently greeted with "Michael Jackson - Beat It" after clicking on the source "REFLEC BEAT VOLZZA".
Narrowing down on the exact search makes for a more intuitive navigation experience, in my opinion.

https://github.com/user-attachments/assets/ca6fe685-2567-414e-ac56-af13dedcc0a8

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30400
